src/view/vueView.js

```diff
@@ -21,30 +20,30 @@ export const  vueRenderView: IRenderView = (data, div) => {
                 Roll
             </button>
         </div>`,
-        data: () => ({ diceValue: 1 }),
+        data: () => ({ diceValue: dice.get("value") }),
```

Comment:
Same here. Add a const. Also let's be consistent with our quotes though a file.

Reply:
yeah. set up a single key in the app and import that key into each view